main.py

```diff
@@ -4,8 +4,33 @@
 import datetime
 import os.path
 import stat
+import time
+import json
+
 THIS_FILE_PATH = os.path.dirname(os.path.realpath(__file__))
 
+saveData = [{
```

Comment:
Since this is test data and you didn't have any piece of the python section you should remove the changes for this file.